library/alloc/tests/str.rs

```diff
@@ -1826,6 +1826,19 @@ fn to_lowercase() {
     assert_eq!("Α'Σ".to_lowercase(), "α'ς");
     assert_eq!("Α''Σ".to_lowercase(), "α''ς");
 
+    assert_eq!("aΣ".to_lowercase(), "aς");
+    assert_eq!("a'Σ".to_lowercase(), "a'ς");
+    assert_eq!("a''Σ".to_lowercase(), "a''ς");
+
+    assert_eq!("ÄΣ".to_lowercase(), "äς");
+    assert_eq!("ä'Σ".to_lowercase(), "ä'ς");
+    assert_eq!("ä''Σ".to_lowercase(), "ä''ς");
+
+    // input lengths around the boundary of the chunk size used by the ascii prefix optimization
+    assert_eq!("abcdefghijklmnoΣ".to_lowercase(), "abcdefghijklmnoς");
+    assert_eq!("abcdefghijklmnopΣ".to_lowercase(), "abcdefghijklmnopς");
```

Comment:
This test was already added a few lines below :) https://github.com/rust-lang/rust/blob/b03d93962816fd82afb619e0cf2083dc67e218e8/library/alloc/tests/str.rs#L1869

Reply:
Thanks, I moved those boundary tests together and removed some other redundant ones.